tests/unit/plugins/v2/test_cmake.py

```diff
@@ -90,8 +90,8 @@ class Options:
                     "-DCMAKE_INSTALL_PREFIX=/foo "
                     '-DCMAKE_SPACED_ARGS="foo bar" '
                     '-DCMAKE_USING_ENV="$SNAPCRAFT_PART_INSTALL"/bar',
-                    'cmake --build . -- -j"${SNAPCRAFT_PARALLEL_BUILD_COUNT}"',
-                    'cmake --build . --target install -- DESTDIR="${SNAPCRAFT_PART_INSTALL}"',
+                    'cmake --build . -j"${SNAPCRAFT_PARALLEL_BUILD_COUNT}"',
+                    'cmake --install . --prefix="${SNAPCRAFT_PART_INSTALL}"',
```

Comment:
`--install` does seem to be the right way.

It seems to me that we should be using DESTDIR based on:
https://cmake.org/cmake/help/v3.18/variable/CMAKE_INSTALL_PREFIX.html#variable:CMAKE_INSTALL_PREFIX

But I'm a cmake newb, so feel free to dispute :D